components/remote-environment-broker/internal/broker/provision_test.go

```diff
@@ -269,6 +272,90 @@ func TestProvisionErrorOnCreatingEventActivation(t *testing.T) {
 	}
 }
 
+func TestProvisionAlreadyExistsErrorOnCreatingEventActivation(t *testing.T) {
```

Comment:
you test is more like a copy of the `TestProvisionErrorOnCreatingEventActivation` please make that more DRY or even consider to do the table test.

Reply:
I did table test for all cases.